config/end-to-end.ini

```diff
@@ -11,22 +11,24 @@ case=extract_and_output
 extract_options=--no-vss
 output_file=extract_and_output1.log
 output_format=dynamic
-reference_output_file=test_data/end_to_end/extract_and_output1.log
+reference_output_file=test_data/end_to_end/extract_and_output.log
 source=test_data/vsstest.qcow2
 
 [extract_and_output2]
 case=extract_and_output
 extract_options=--no-vss --text C:
 output_file=extract_and_output2.log
```

Comment:
Please change this to match the reference_output_file name

Reply:
It is actually easier to have the file named corresponding to the test, so it is clear in the test result directory that this file belongs to the extract_and_output2 test